docs/Toy/Toy_Ch2.ipynb

```diff
@@ -392,8 +392,14 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
Can you please clear this metadata? The `Makefile` has a rule that shows how this is done.